include/seqan3/core/algorithm/detail/execution_handler_parallel.hpp

```diff
@@ -90,7 +90,7 @@ class execution_handler_parallel
     }
 
     //!\brief Constructs the execution handler spawning std::thread::hardware_concurrency many threads.
-    execution_handler_parallel() : execution_handler_parallel{std::thread::hardware_concurrency()}
+    execution_handler_parallel() : execution_handler_parallel{1u}
```

Comment:
Please correct the `\brief` documentation and add (e.g. in \detail) the reasoning you wrote in this PR why we set the default to 1.

Reply:
I added it. I didn't add the sentence about live/deadlock since it is a bug and should be fixed soon™